gdb-9.2/sim/common/sim-core.c:75:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-core.c:77:7: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-core.c:375:6: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-core.c:376:4: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-cpu.c:64:3: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-engine.c:83:7: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/sim-engine.c:125:7: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/sim-engine.c:132:7: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/sim-io.c:299:5: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/sim-module.c:225:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-module.c:235:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-module.c:245:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-module.c:255:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-module.c:265:2: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-module.c:269:3: warning: incompatible implicit declaration of built-in function 'free'
gdb-9.2/sim/common/sim-reason.c:42:7: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/nrun.c:60:7: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:90:5: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:94:7: warning: incompatible implicit declaration of built-in function 'abort'
gdb-9.2/sim/common/nrun.c:119:4: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:125:4: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:134:5: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:150:7: warning: incompatible implicit declaration of built-in function 'exit'
gdb-9.2/sim/common/nrun.c:239:3: warning: incompatible implicit declaration of built-in function 'exit'

--- gdb-9.2/sim/common/sim-basics.h
+++ gdb-9.2/sim/common/sim-basics.h
@@ -35,6 +35,7 @@
 
 #include <stdarg.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <setjmp.h>
 
 #ifdef __CYGWIN32__
